libs/tex/poisson_blending.cpp

```diff
@@ -52,14 +63,14 @@ poisson_blend(mve::FloatImage::ConstPtr src, mve::ByteImage::ConstPtr mask,
 
     assert(src->width() == mask->width() && mask->width() == dest->width());
     assert(src->height() == mask->height() && mask->height() == dest->height());
-    assert(src->channels() == 3 && dest->channels() == 3);
+//    assert(src->channels() == 3 && dest->channels() == 3);
```

Comment:
```suggestion
      assert(dest->channels() >= 3 && dest->channels() == src->channels());
```
In light of line 73, though, maybe just `assert(dest->channels() > 0);`?